web/src/store/modules/usePluginStore.ts

```
@@ -51,20 +50,20 @@
       return plugins.value as keyPlugins
     }
 
-    async function enabled(pluginName: string): boolean {
+    async function enabled(pluginName: string) {
       if (plugins.value[pluginName]) {
-        const plg = plugins.value[pluginName]
+        const plg = plugins.value[pluginName] as Plugin.PluginConfig
         plg.config.enable = true
         if (plg?.hooks && plg?.hooks?.start) {
           await plg.hooks.start(plg.config)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)